### ISSUE TITLE
[Bug] After updating, no error thrown in terminal, but Video Combine node is highlighted in red and fails silently. 

### ISSUE THREAD

**Issue by ricardofeynman** [Kosinkadink/ComfyUI-VideoHelperSuite#99]
I'm unfortunately not certain which commit I updated from, but I think I had previously updated as recently as yesterday and Video Combine was still working for me without issue earlier today. 

I'm unsure what else to report as there is no indication of the problem, the node just highlights in red and then the workflow process completes without Video Combine creating the video. 

Any ideas?

**Comment by Kosinkadink** [comment]
If it highlights red, check the console for error messages after pressing Queue Prompt. And also try to place a new Video Combine node, or refreshing the page.

**Comment by AustinMroz** [comment]
From your description, it sounds like it's not an error in the python code failing, but from the workflow migration code. This can occur in 2 cases to my knowledge, if a major error prevents the script from loading entirely (unlikely, should only occur with syntax errors during development) or if  there's very old state in the migration code. In both cases, there should have been an error message in the webui explaining that simply removing and re-adding the node should fix things.

Since you've updated recently, there likely is an underlying bug that can be fixed here.

**Comment by ricardofeynman** [comment]
Thanks for the speedy replies. Unfortunately there are zero indicators in the terminal. I had tried all relevant suggested approaches prior to opening an issue, my bad for lack of prior elucidation. 

I am running a heavy install, all custom nodes avaliable via the manager installed, except those that are only relating to interfacing with commercial services. There's also many nodes I've had to disable because of package version conflicts, so I'm very keenly checking the terminal readouts during any updates to spot those errors that can throw once and then don't show again subsequently. Nothing obvious caught my eye.

Some discussions from another issue https://github.com/Kosinkadink/ComfyUI-VideoHelperSuite/issues/80 has perhaps given me a clue. I switched to using miniConda yesterday and so had to manually reinstall all custom node-relevant site-packages for the conda python environment. There was at one stage a message about imageio ffmpeg, Searching for ffmpeg in the terminal after having pip installed ffmpeg, the imagio ffmpeg message no longer appears but presently a warning message from WAS nodes states the following:

WAS Node Suite Warning: `ffmpeg_bin_path` is not set in `/home/x/Documents/_ComfyUI_2023-11-25_1735C/ComfyUI/custom_nodes/was-node-suite-comfyui/was_suite_config.json` config file. Will attempt to use system ffmpeg binaries if available.

I'm going to go investigate if the switch to Conda may be what is causing the Video Combine silent fail due to an ffmpeg path issue.

**Comment by QTIGER2** [comment]
When i update comfyui , I also met this problem, My loadvideo node and videocombine node are red now. And in the comfyui manager,this extension indicate "import failed"!!!

**Comment by ricardofeynman** [comment]
nvm, I got it working again.

Not ffmpeg path issue apparently. In my case I moved all custom nodes that I'd updated in the last couple days to another folder and copied them back in a few at a time checking if the Video Combine issue I was experiencing earlier could be pinpointed to a conflict with another recently updated custom node set, but I ended up going through all of them and couldn't get a clear picture of what was causing it. ¯\_(ツ)_/¯
